snap/snapcraft.yaml

```diff
@@ -117,10 +117,13 @@ parts:
       source scripts/activate.sh
       set -u
 
-      cd examples/lighting-app/linux/
+      cd examples/lighting-app/lighting-common/
+      cp -vr $CRAFT_PART_BUILD/lighting-common/* .
+
+      cd ../linux/
 
       # Copy and replace the application files
-      cp -vr $CRAFT_PART_BUILD/* .
+      cp -vr $CRAFT_PART_BUILD/linux/* .
```

Comment:
These are getting confusing to follow. How about not changing directories and simply copying from sources to targets (2 commands)?

The comment should be moved to top.